cli/src/commands/create/dataset.rs

```diff
@@ -27,9 +27,9 @@ pub struct CreateDatasetArgs {
     /// Names or ids of the sources in the dataset
     sources: Vec<SourceIdentifier>,
 
-    #[structopt(short = "e", long = "entity-kinds")]
-    /// Entity kinds to enable at dataset creation (pass one flag for each kind)
-    entity_kinds: Vec<EntityKind>,
+    #[structopt(short = "e", long = "entity-def")]
```

Comment:
Does it accept multiple values space-separated?

Reply:
Nope, it does not. I did try to add a struct called `EntityDefs` which was just made of a single `Vec<EntityDef>` but then the user would have to provide a json of the form
```
{
    "entity_defs": { ... }
}
```
which I found cumbersome, and I'd need to remove this layer before passing to the api, so I decided to go for the option of providing one flag per entity def. Happy to pair to see how to make it work though!

Reply:
You probably needed this structure:

```rust
#[derive(Deserialize)]
struct EntityDefs(Vec<EntityDef>);
```

... which I think should deserialize exactly as `[entity_def_1, entity_def_2, ...]` without the wrapping object.

Reply:
It worked, thanks!